### PR TITLE
Feature/replace header navigation

### PR DESCRIPTION
**Pull request by ivo-ritense**

### CODE REVIEW

File: core/src/main/java/com/ritense/valtimo/web/rest/ChoiceFieldResource.java [hunk 1]
```diff
@@ -81,14 +81,25 @@ public ResponseEntity<ChoiceField> updateChoiceField(@Valid @RequestBody ChoiceF
...
+    @Deprecated(since = "12.0.0", forRemoval = true)
     public ResponseEntity<List<ChoiceField>> getAllChoiceFields(Pageable pageable) {
         logger.debug("REST request to get a page of ChoiceFields");
         Page<ChoiceField> page = choiceFieldService.findAll(pageable);
         HttpHeaders headers = PaginationUtil.generatePaginationHttpHeaders(page, "/api/v1/choice-fields");
         return new ResponseEntity<>(page.getContent(), headers, HttpStatus.OK);
     }
 
+    @GetMapping("/v2/choice-fields")
+    public ResponseEntity<Page<ChoiceField>> getAllChoiceFieldsPaged(Pageable pageable) {
```
Comment (marijnritense) [inline core/src/main/java/com/ritense/valtimo/web/rest/ChoiceFieldResource.java:96]: missing security config for the new endpoints

File: core/src/main/java/com/ritense/valtimo/web/rest/ProcessResource.java [hunk 2]
```diff
@@ -83,18 +95,6 @@
...
-import java.time.LocalDateTime;
-import java.time.ZoneId;
-import java.util.Date;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Optional;
-import java.util.stream.Collectors;
 import static com.ritense.valtimo.camunda.repository.CamundaProcessDefinitionSpecificationHelper.VERSION;
```
Comment (theo-ritense) [inline core/src/main/java/com/ritense/valtimo/web/rest/ProcessResource.java]: These static imports should be on top according to the new Checkstyle settings.

**Comment by theo-ritense** [comment]
@ivo-ritense Can you resolve the conflicts?

File: core/src/main/java/com/ritense/valtimo/security/config/ProcessHttpSecurityConfigurer.java [hunk 3]
```diff
@@ -56,7 +56,8 @@ public void configure(HttpSecurity http) {
                 .requestMatchers(antMatcher(GET, "/api/v1/process/{processInstanceId}/xml")).authenticated()
                 .requestMatchers(antMatcher(GET, "/api/v1/process/{processInstanceId}/activities")).authenticated()
                 .requestMatchers(antMatcher(GET, "/api/v1/process/{processInstanceId}/comments")).authenticated()
-                .requestMatchers(antMatcher(GET, "/api/v1/process/{processDefinitionName}/search")).authenticated()
+                .requestMatchers(antMatcher(POST, "/api/v1/process/{processDefinitionName}/search")).authenticated()
```
Comment (Klaas-Ritense) [inline core/src/main/java/com/ritense/valtimo/security/config/ProcessHttpSecurityConfigurer.java:59]: Changing the GET to a POST. `/v1` shouldn't be changed

**Review by Klaas-Ritense — APPROVED** [review]

**Pull request merged**